website/docs/reference/resource-configs/lookback.md

```diff
@@ -3,14 +3,14 @@ title: "lookback"
 id: "lookback"
 sidebar_label: "lookback"
 resource_types: [models]
-description: "dbt uses `lookback` to detrmine how many 'batches' of `batch_size` to reprocesses when a microbatch incremental model is running incrementally."
+description: "dbt uses `lookback` to determine how many 'batches' of `batch_size` to reprocess when a microbatch incremental model is running incrementally."
 datatype: int
 ---
 
 <VersionCallout version="1.9" />
```

Comment:
Doesn't need to be addressed with this PR but this renders as "Available from dbt v1.9 or with the dbt Cloud "Latest" release track" but the "Compatible" release track is v1.9, so wouldn't it be available there too? Maybe I'm overanalyzing, but it seems confusing
```suggestion
<VersionCallout version="1.9" />
```